### PR TITLE
create login screen

### PR DESCRIPTION
**Pull request by sengayire**
#### What does this PR do?
- build a login screen 
- redirect the user to his/her profile after login successfully
#### Description of Task to be completed?
- User will be able to login using Gmail Account
- User will be redirected to a profile screen after login successfully
- user will be able to logout 
#### How should this be manually tested?
- Clone repo local machine
- After cloning a repo, cd into CodeLab directory
- in your terminal run the app with ```expo start``` or ```yarn start```
- scan the QR code with your expo client (local device)
- Or run it in your simulator by pressing `i` for ios and `a`  android .
#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
- N/A
#### Screenshots (if appropriate)

Screen Recording 2019-08-27 at 14 30 25 2019-08-27 19_08_20

#### Questions:
-N/A

### CODE REVIEW

File: src/screens/ProfileScreen.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+} from 'react-native';
+import * as AppAuth from 'expo-app-auth';
+
+const ProfileScreen = ({ data, response, navigation }) => {
+  signOutAsync = async response => {
+    const config = {
+      issuer: 'https://accounts.google.com',
+      scopes: ['profile'],
+      clientId:
+        '687916247467-6dt3274n9544kfk53udm2di107vvmahe.apps.googleusercontent.com'
```
Comment (abayo-luc) [inline src/screens/ProfileScreen.js]: This should be managed in .env

File: src/screens/ProfileScreen.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+      <TouchableOpacity
+        onPress={() => signOutAsync(response && response.accessToken)}
+      >
+        <Text>Logout</Text>
+      </TouchableOpacity>
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
```
Comment (abayo-luc) [inline src/screens/ProfileScreen.js]: Kindly separate CSS from the components.

File: src/screens/ProfileScreen.js (hunk 1 above)
Comment (sengayire) [reply src/screens/ProfileScreen.js]: @abayo-luc done

File: src/screens/ProfileScreen.js (hunk 2 above)
Comment (sengayire) [reply src/screens/ProfileScreen.js]: @abayo-luc done

File: .env [hunk 3]
```diff
@@ -0,0 +1,3 @@
+ISSUER_URL = 'https://accounts.google.com'
```
Comment (abayo-luc) [inline .env]: This should be in .gitignore

File: package.json [hunk 4]
```diff
@@ -8,11 +8,16 @@
     "eject": "expo eject"
   },
   "dependencies": {
+    "babel-plugin-inline-dotenv": "^1.3.3",
```
Comment (abayo-luc) [inline package.json]: are you still using this dependency?

**Pull request merged**